t/e2e/rpc/rpc_test.go

```diff
@@ -52,7 +52,7 @@ func (s *RPCTestSuite) TestCallRPC() {
 
 		s.NoError(s.StatusNode.Start(nodeConfig))
 
-		rpcClient := s.StatusNode.RPCClient()
+		rpcClient := s.StatusNode.RPCPrivateClient()
```

Comment:
As this test is called `TestCallRPC`, we can change its name to `TestCallPrivateRPC` and create another one for testing `CallRPC` (the latter thing as a separate issue probably).